.github/workflows/build.yml

```diff
@@ -50,6 +50,40 @@ jobs:
         name: test-results
         if-no-files-found: error
         path: '**/target/*-reports/*.xml'
+  build-launcher-win64:
+    runs-on: windows-2019
```

Comment:
What about `windows-latest`?

Reply:
Again, I found something that indicated that windows-2019 was the one including Visual Studio 2019. Maybe windows-latest contains the same version of Visual Studio or a version that would be equally good..